.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+sudo: false
+python:
+  - "3.3"
+addons:
+  postgresql: "9.3"
+before_script:
+  - psql -c 'create database feedthefox_db;' -U postgres
+install:
+  - pip install -r requirements.txt
```

Comment:
I would prefer to use peep instead:
Also check:
https://github.com/mozilla/feedthefox/blob/master/tox.ini#L17

It looks like we are installing requirements there.